docs/tools/vdb_table/data/usearch.json

```diff
@@ -53,29 +53,29 @@
     "comment": "using the multi param in index creation"
   },
   "sparse_vectors": {
-    "support": "",
+    "support": "partial",
```

Comment:
please provide a reference for this

Reply:
Can you define sparsity? Some people consider it to be a sparse layout of vectors, others consider it to be the ability to definitively describe the meaning of a specific "dimension" of the feature vector. In both cases, the answer will be - yes, but the links will be different. Here is one example of a sparse-search project built with USearch: https://registry.opendata.aws/usearch-molecules/

Reply:
Sparse Vector Support is the ability to store high-dimensional vectors like those generated by bag-of-words, TF-IDF, BM25, SPLADE on a large vocabulary, and use them for retrieval.

Reply:
The molecules example you've linked to seems to encode fixed length vectors of relatively low dimensions (<=2048), so I assume they are being stored and retrieved with conventional dense vector techniques.